com.reprezen.swagedit.core/src/com/reprezen/swagedit/core/schema/ComplexTypeDefinition.java

```diff
@@ -139,4 +140,9 @@ public boolean validate(AbstractNode valueNode) {
 
         return isValid;
     }
+    
+    @Override
+    public String getLabel() {
+        return Iterables.getFirst(complexTypes, null).getLabel();
```

Comment:
You will get a NPE if complexTypes is null here.

Reply:
Thanks! Addressed in  9f2cd7d